torchvision/transforms/functional.py

```diff
@@ -1578,3 +1608,21 @@ def elastic_transform(
     if not isinstance(img, torch.Tensor):
         output = to_pil_image(output, mode=img.mode)
     return output
+
+
+# TODO in v0.17: remove this helper and change default of antialias to True everywhere
+def _check_antialias(antialias: Optional[Union[str, bool]], interpolation: InterpolationMode) -> Optional[bool]:
+    if isinstance(antialias, str):  # it should be "warn", but we don't bother checking against that
+        # TODO Should we only warn if input is a tensor, or for PIL images too?
```

Comment:
Thoughts?

Reply:
I would only warn for tensors. Since the value is, and will be, ignored for PIL and we already warn that it is passed in the first place 

https://github.com/pytorch/vision/blob/135a0f9ea9841b6324b4fe8974e2543cbb95709a/torchvision/transforms/functional.py#L474-L476

the warning here is irrelevant for the user.

Reply:
OK, we're only warning for Tensors now. One minor caveat is that since we can't warn in `__init__()` of `Resize()` anymore, the warning will only happen during `forward()`. I assume that's fine? Right now, I tend to prefer that to the alternative of warning for PIL users who should not have to worry about all this